service/worker/deletenamespace/deleteexecutions/activities.go

```diff
@@ -41,14 +41,23 @@ import (
 	"go.temporal.io/server/common/quotas"
 )
 
+var activityHandles = &Activities{LocalActivities: &LocalActivities{}}
+
 type (
 	Activities struct {
+		*LocalActivities
```

Comment:
I feel like ideally we should separate those two. Is there any implementation that got invoked both as local activity and normal activity?